.changeset/long-mayflies-yell.md

```diff
@@ -0,0 +1,5 @@
+---
+"@changesets/action": minor
+---
+
+Add `pullRequestNumber` to the output of the action
```

Comment:
```suggestion
Added `pullRequestNumber` to the action's outputs
```

Reply:
I've wanted to commit this suggestion through the GitHub interface - but it seems like I don't have sufficient permissions to push to your fork.

You can check how to allow this for the future PRs here - https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork . At the moment just commit this suggestion with a click on a button.